Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md

```diff
@@ -10,6 +10,18 @@ On Umbraco Cloud, users work in almost the same way as on a normal installation
 
 In this article, we will show how users work, as well as explain the different settings for users on Umbraco Cloud.
 
+## Umbraco ID
+
+On Umbraco CLoud project we use Umbraco ID for new projects and projects on 8.9.1 and above.
+
+Umbraco ID is a centralised login for all users on Umbraco Cloud, both team members and Umbraco Backoffice users.
+It is used when you log in to the Umbraco Cloud Portal, projects,
+as well as when you clone down a project to your local machine.
+
+Because of this there are no longer any user files added in the data folder when a user is invited to a specific project.
+
+Instead when login in to the Umbraco backoffice locally you will be promted to log in using Umbraco ID.
```

Comment:
Should we also mention that it is possible to create local users, if you need to work with the site while you're offline, for some reason?
Not sure how relevant this would be 🤔

Reply:
This sentance is a bit awkward 
>Instead when login in to the Umbraco backoffice locally you will be promted to log in using Umbraco ID.

How about: "When working locally the initial login will go through Umbraco Id and the online login flow. After the intial login you can set a password on your user or create a new login for the backoffice, which can be used for local logins."

Reply:
Good suggestion, i will replace the it :)